clang/www/analyzer/index.html

```diff
@@ -69,48 +69,7 @@
 
 <h1>Clang Static Analyzer</h1>
 
-<p>The Clang Static Analyzer is a source code analysis tool that finds bugs in
-C, C++, and Objective-C programs.</p>
-
-<p>Currently it can be run either from the <a href="command-line.html">command
-  line</a> or if you use macOS then <a href="xcode.html">within Xcode</a>. When
-invoked from the command line, it is intended to be run in tandem with a build
-of a codebase.</p>
-
-<p>The analyzer is 100% open source and is part of the <a
-href="https://clang.llvm.org">Clang</a> project. Like the rest of Clang, the
-analyzer is implemented as a C++ library that can be used by other tools and
-applications.</p>
```

Comment:
The intro part can probably stay? (Except the second paragraph.)

Reply:
I have re-added the relevant parts to the introduction part.
The flow of text in the HTML is even more aesthetically pleasing this way 👍